integration-tests/narayana-jta/src/main/java/io/quarkus/narayana/jta/TransactionBeanWithEvents.java

```diff
@@ -0,0 +1,134 @@
+package io.quarkus.narayana.jta;
+
+import java.util.concurrent.atomic.*;
+
+import javax.enterprise.context.*;
+import javax.enterprise.context.spi.*;
+import javax.enterprise.event.*;
+import javax.enterprise.inject.spi.*;
+import javax.inject.*;
+import javax.transaction.*;
+
+import org.jboss.logging.*;
+
+@ApplicationScoped
+public class TransactionBeanWithEvents {
+    private static final Logger log = Logger.getLogger(TransactionBeanWithEvents.class);
+
+    private static AtomicInteger initializedCount = new AtomicInteger(0);
```

Comment:
These events are fires synchronously, hence you shouldn't need `AtomicInteger` for it.
Unless you mix that with some propagation, but that doesn't seem to be the case here.

Reply:
ok. I have no issue to change it to `int`